_includes/page-core/breadcrumbs.html

```diff
@@ -2,19 +2,29 @@
 {% assign link='/' %}
 <ol class="breadcrumb" vocab="http://schema.org" typeof="BreadcrumbList">
     {% assign position = 1 %}
+    {% assign current_dir = site.data.breadcrumbs %}
+
     {% for dir in parent_dirs %}
         {% assign link=link | append: dir | append: '/' %}
+
+        {% assign humanized_name = dir %}
```

Comment:
I think I would rather call this `pretty_name` instead of `humanized_name`. It is quite common to see the term "pretty name" in other toolkits, so I already have a good idea of what it means.